app/wama.web/Controllers/UserToolController.cs

```diff
@@ -236,7 +236,8 @@ public IActionResult AddCertification(string memberId)
         public async Task<IActionResult> ViewCertifications(string memberId)
         {
             SetActiveConsoleTool(Constants.ADMIN_CONSOLE_USERS_PATRONS);
-            var Certifications = await _CertificationService.GetCertificationsAsync(memberId);
+            var Certifications = await _UserAccountService.GetUserAccountAsync(memberId);
```

Comment:
But, things can be improved.

Why are you setting a variable named **_Certifications_** to a **_UserAccount._**?